src/main/java/Duke.java

```diff
@@ -1,10 +1,112 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Naruto and one day I will be Hokage! \nWhat can i do for you?");
+        Scanner scanner = new Scanner(System.in);
+        Naruto naruto = new Naruto();
+        String lineBreak = "-".repeat(20);
+
+        boolean breakLoop = false;
+        while (!breakLoop) {
+            String rawInput = scanner.nextLine().trim();
+            String[] input = rawInput.split(" ", 2);
+            System.out.println(lineBreak);
+            switch (input[0]) {
+                case "list":
```

Comment:
Perhaps it will look neater to follow the switch-case indentation (same-level) from the coding standards?